lib/parsers.js

```diff
@@ -14,11 +14,11 @@ var stacktrace = require('error-stack-parser')
 
```

Comment:
I see we've modified this module to use the "export everything at the end" style, changed function definitions to be in the `function functionName` style, and changed the formally local-to-this-module `getModule` to `_moduleNameFromFrames` and are now exporting this function in order to test it.

Reply:
@astorm I hope you are good with those changes. 

- I noticed you and I both perhaps gravitated to that style for module.exports (from recently added lib/logging.js and lib/wildcard-matcher.js).
- The `fooFromBar` is naming style if grown fond of. The influence for me was https://www.joelonsoftware.com/2005/05/11/making-wrong-code-look-wrong/